build

```diff
@@ -22,9 +22,13 @@ export CGO_ENABLED=1
 echo "Building ${NAME}..."
 go build -ldflags "${GLDFLAGS}" -o ${GOBIN}/${NAME} ${REPO_PATH}/internal
 
-for D in tests/stubs/*; do
-	if [ -d "${D}" ]; then
-		echo "Building ${D}"
-		go install ${REPO_PATH}/${D}
-	fi
-done
+if [ "$GOHOSTARCH" == "$GOARCH" ]; then
+	# Don't try to go install things when cross compiling because
+	# go install disallows it.
```

Comment:
Would it make more sense to run `go build` when cross compiling, to at least check that these binaries compile?